WordPress/src/main/java/org/wordpress/android/ui/posts/EditorBloggingPromptsViewModel.kt

```diff
@@ -15,7 +16,9 @@ import javax.inject.Named
 class EditorBloggingPromptsViewModel
 @Inject constructor(
     private val bloggingPromptsStore: BloggingPromptsStore,
-    @Named(BG_THREAD) private val bgDispatcher: CoroutineDispatcher
+    @Named(BG_THREAD) private val bgDispatcher: CoroutineDispatcher,
```

Comment:
Suggestion/preference: no need to change if it doesn't make sense, but I prefer having the `CoroutineDispatchers` as the last parameters as they are related to Kotlin-specifics rather than business logic.